src/qtt/algorithms/random_telegraph_signal.py

```diff
@@ -73,21 +73,26 @@ def _plot_rts_histogram(data, num_bins, double_gaussian_fit, split, figure_title
     _, bins, _ = plt.hist(data, bins=num_bins)
     bincentres = np.array([(bins[i] + bins[i + 1]) / 2 for i in range(0, len(bins) - 1)])
 
-    plt.plot(bincentres, double_gaussian(bincentres, double_gaussian_fit), 'r', label='Fitted double gaussian')
+    left_gaussian = [double_gaussian_fit[i] for i in [4, 2, 0]]
```

Comment:
The wrong data format is chosen for this structure. These indexes are not clear for the reader. It const me some time to figure out what is happening. Instead of rebuilding we earlier did something like:

    get_left_mean_std_amplitude = operator.itemgetter(4, 2, 0)
    get_right_mean_std_amplitude = operator.itemgetter(5, 3, 1)
    left_gaussian = list(get_left_mean_std_amplitude(double_gaussian_fit))
    right_gaussian = list(get_right_mean_std_amplitude(double_gaussian_fit))

Reply:
That is a reasonable solution yes. Without backwards compatibility we would consider different formats